content/en/users/cloud-compute/ec3/apps/ecas/_index.md

```diff
@@ -120,7 +120,7 @@ To access the front-end of the cluster:
   authentication.
 
 ```shell
-[fabrizio@MBP EC3]$ ssh -i key.pem cloudadm@134.158.151.218
+[user@localhost EC3]$ ssh -i key.pem cloudadm@134.158.151.218
```

Comment:
Here it would be better to use a fake IP

Reply:
I'll fix it